test/parallel/test-net-connect-handle-econnrefused.js

```diff
@@ -27,11 +27,10 @@ const assert = require('assert');
 const server = net.createServer();
 server.listen(0);
 const port = server.address().port;
-const c = net.createConnection(port);
-
-c.on('connect', common.mustNotCall());
-
-c.on('error', common.mustCall((e) => {
-  assert.strictEqual('ECONNREFUSED', e.code);
-}));
-server.close();
+server.close(() => {
```

Comment:
This test seems a bit racey in general. If you wait for the server to close, couldn't another process potentially take the port?

Reply:
@cjihrig Hmm... You're right.
I think better is move `test-net-connect-handle-econnrefused.js` to `test/sequential`.
If this test run sequentially, another test cannot take the port, right?

Reply:
I would stick to having it be in parallel and seeing if this fixes it. We need less tests in sequential, not more, as they take a really long time to run.